src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
OK, I was just about to make this edit, when I realized you're right.  I did indeed make this the "Max" power, and that was intentional (at least at the time).  I did that so that the number wasn't bouncing every single update, because if you can't read the value it doesn't do any good.  Imagine the display rapidly changing between "0.2", "0.01", and "0.000001".

In that case, while we can't do a tool tip we can modify the string to convey the meaning.  Maybe "kOS Max Power".  I'm also going to add `guiUnits = "EC/s"` to the definition, and round the display value to make it more readable.

I apologize for the confusion.  I wrote the code a bit before I actually submitted the PR and it wasn't until I dug into the code that I recognized what I did and why.